### PR TITLE
Restructure Docker flow

### PR DESCRIPTION
**Pull request by yevr19**
Add new docker-compose.services.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    image: 'online-store/online-store:latest'
+    container_name: online-store-app
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - '8083:8083'
+    user: root
+    environment:
+      AWS_ENDPOINT_URL: http://awslocal:4566
```
Comment (Sunagatov) [inline docker-compose.yml:22]: You can delete it

File: docker-compose.services.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 version: '3.9'
-services:
 
+networks:
```
Comment (Sunagatov) [inline docker-compose.services.yml:3]: if you move networks there, may be it is a good idea to move volumes there as well?

File: docker-compose.services.yml [hunk 3]
```diff
@@ -65,15 +39,11 @@ services:
...
+      - dev
 
 volumes:
   sonarqube_conf:
   sonarqube_data:
   sonarqube_extensions:
   sonarqube_bundled-plugins:
-  db:
\ No newline at end of file
+  db:
```
Comment (Sunagatov) [inline docker-compose.services.yml:49]: extra space below

**Review by Sunagatov — COMMENTED** [review]
Reviewed

File: docker-compose.services.yml (hunk 3 above)
Comment (yevr19) [reply docker-compose.services.yml:49]: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

File: docker-compose.services.yml (hunk 2 above)
Comment (yevr19) [reply docker-compose.services.yml:3]: no it's a common practice to leave volumes beneath the file

**Review by Reyzis2021 — APPROVED** [review]

**Review by vitaliibredun — APPROVED** [review]
Approved

**Review by Sunagatov — APPROVED** [review]

**Pull request merged**